src/Exceptions/TwitterException.php

```diff
@@ -34,7 +34,9 @@ public static function fromPayload(array $payload): TwitterException
             // For most errors related to rule creation, details are provided.
             $hasDetails = array_key_exists('details', $error);
 
-            $buffer .= sprintf("[%s] %s %s [%s]\n", $error['title'], $error['value'], $hasDetails ? ': ' . implode('; ', $error['details']) : '', $error['id']);
+            $errorIdOrType = $error['id'] ?? ($error['type'] ?? '');
```

Comment:
If there's no ID or type, that would leave empty brackets on the error message. To solve that, perhaps change the default value to "unknow error" or something.

It shouldn't happen but, Twitter also returns 2xx on errors so what do we know...